packages/react/src/components/organisms/Header/_index.scss

```diff
@@ -0,0 +1,19 @@
+@use "~normalize-scss/sass/normalize";
+@use '../../molecules/HeaderSearch';
+@use '../../molecules/MainNav';
+@use "00-base/elements";
+@use "00-base/layout";
+@use "00-base/fonts";
+@use '01-atoms/decorative-link';
+@use '01-atoms/button-with-icon';
+@use '01-atoms/button-search';
+@use '01-atoms/input-typeahead';
+@use '01-atoms/site-logo';
```

Comment:
Ideally, I think these scss for SiteLogo, HeaderSearch, UtilityNav scss should be imported in the respective index.scss for those specific components and then imported here. So that the lower-level component scss indexes are included in the upper-level component scss indexes

Reply:
I think that the contents of this _index.scss file should live in assets as 03-organisms/header.scss. Then, this file should only be:
```
@use '03-organisms/header';
```
We should not be combining `_index.scss` files from the react package together to make a story's rendered and displayed css files. Instead, we should be pulling scss files from the assets package into `_index.scss` for each story.